File: 01.JAVA/pg-226-230/pg_226_02.java
import java.util.Scanner;

public class pg_226_02
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num, i;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ AN INTEGER AND DISPLAY ALL EVEN NUMBERS BETWEEN ONE AND THE GIVEN NUMBER");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");
        
        System.out.print("Write an integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }
/*
        for(i=1; i<=num; i++)
        {
            if(i % 2 == 0)
            {
                System.out.print(i + ", ");
            }
        }
*/
        i= 1;
        while(i <= num)
        {
            if(i % 2 == 0)
            {
                System.out.print(i + "-");
            }
        i++;
        }
    }
}

File: 03.JAVASCRIPT/pg-226-230/pg_226_02.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJ");
console.log("READ AN INTEGER AND DISPLAY ALL EVEN NUMBERS BETWEEN ONE AND THE GIVEN NUMBER");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJ");
console.log("");

let numInput = document.querySelector("#num");
let bntProof = document.querySelector("#btnProof");
let textar = document.querySelector("#textar");

let i = 1;

function validateNumber()
{
    let num = Number(numInput.value);

    if(num < 0)
        {
            num *= -1;
        }
/*
    for(i=1; i<=num; i++)
        if(i % 2 == 0)
        {
            textar.value += (i+ ", ");
        }
*/

    while(i <= num)
    {
        if(i % 2 == 0)
        {
            textar.value += (i+ "-");
        }
        i++;
    }
}
btnProof.addEventListener("click", validateNumber);
numInput.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});